tools/ci_build/github/windows/setup_env_cuda_11.bat

```diff
@@ -1,2 +1,6 @@
+REM Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
We should try to deprecate this file, because it mixes CUDA 11.1 and CUDA 11.4 DLLs together.